uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -1277,7 +1284,7 @@ sub get_mincs {
     my @files_list;
     foreach my $file (@files) {
         next unless $file =~ /\.mnc(\.gz)?$/;
-        my $cmd= "Mincinfo_wrapper -quiet -tab -file -date $this->{TmpDir}/$file";
+        my $cmd= "Mincinfo_wrapper -quiet -tab -file -attvalue acquisition:acquisition_id $this->{TmpDir}/$file";
```

Comment:
As discussed, we should use `quotemeta` on all arguments that might contain shell special characters to avoid bad surprises.

Reply:
done in the next commit